crates/edr_gas_report/src/gas_report.rs

```diff
@@ -1,7 +1,13 @@
+use std::fmt::Display;
```

Comment:
Polish: Since the crate only consists of one file, feel free to make this a single-file crate with just a `lib.rs`; i.e. move `gas_report.rs` -> `lib.rs`.